### PR TITLE
Fix data sparsity issue in place page

### PR DESCRIPTION
**Pull request by shifucun**
- Borrow category data from all categories into Overview page.
- For a set of India cities, use the District place as search result.
- [Bonus]: remove additional stat vars for place page data fetch.

### CODE REVIEW

File: static/js/place/place_autocomplete.ts [hunk 1]
```diff
@@ -19,6 +19,53 @@ import { getPlaceDcids } from "../utils/place_utils";
 let ac: google.maps.places.Autocomplete;
 let acs: google.maps.places.AutocompleteService;
 
+// Some India cities have limited data, override to show the corresponding district page.
```
Comment (beets) [inline static/js/place/place_autocomplete.ts:22]: can we put a todo to move this to flask? (so we do the override in the place2dcid call - to catch all our place searches, and to reduce the size of our js).

**Review by beets — APPROVED** [review]
thanks for the fix!

is there also a case where we might have linked to category pages (which are now blank or have been merged into overview) and we would need to redirect? for straggler links to places with categories selected. a lower priority fix, in the grand scheme of things..

File: static/js/place/place_autocomplete.ts (hunk 1 above)
Comment (shifucun) [reply static/js/place/place_autocomplete.ts]: Moved in this PR

**Comment by shifucun** [comment]
> thanks for the fix!
> 
> is there also a case where we might have linked to category pages (which are now blank or have been merged into overview) and we would need to redirect? for straggler links to places with categories selected. a lower priority fix, in the grand scheme of things..

Right now we don't show empty category pages. If there is no data, then the category page will not show up. Now only category with data is merged into the Overview page.

**Comment by shifucun** [comment]
> > > thanks for the fix!
> > > is there also a case where we might have linked to category pages (which are now blank or have been merged into overview) and we would need to redirect? for straggler links to places with categories selected. a lower priority fix, in the grand scheme of things..
> > 
> > 
> > Right now we don't show empty category pages. If there is no data, then the category page will not show up. Now only category with data is merged into the Overview page.
> 
> Ah yes, but for these pages that will be "losing" their category pages, we should redirect back to the overview if there are old links online / bookmarked.. e.g. https://autopush.datacommons.org/place/wikidataId/Q15116?category=Health

Now I see what you mean.. I will bundle this with webdriver tests in a PR

**Comment by beets** [comment]
thank you!!

**Pull request merged**